src/PhpSpec/Formatter/JUnitFormatter.php

```diff
@@ -52,7 +52,7 @@ class JUnitFormatter extends BasicFormatter
         ExampleEvent::SKIPPED => 'skipped',
     );
 
-    public function __construct(PresenterInterface $presenter, IO $io, StatisticsCollector $stats)
+    public function __construct(Presenter $presenter, IO $io, StatisticsCollector $stats)
```

Comment:
the issue is that the PR deprecating PresenterInterface in 2.x did only half of the work (which was not caught because PhpSpec does not have the testing infrastructure in place to detect the usage of deprecated APIs)